aws-iam-role/main.tf

```diff
@@ -6,6 +6,13 @@ locals {
     owner     = var.owner
     managedBy = "terraform"
   }
+  all_saml_idps = concat(
+    var.saml_idps,
+    // NOTE(el): by default, these assume SAML assertions signed for AWS specifically.
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Single line comments should begin with #